index.html

```diff
@@ -33,247 +33,337 @@
-                    class="botcards" alt="An image of a Paper" />
-                <img src="https://static.thenounproject.com/png/88666-200.png" width="200" height="200" class="botcards"
-                    alt="An image of a Scissor" />
-            </div>
+        <div id="bChoices">
+          <img
+            src="https://static.thenounproject.com/png/477914-200.png"
+            width="200"
+            height="200"
+            alt="An Image of a Rock"
+          />
+          <img
+            src="https://static.thenounproject.com/png/477912-200.png"
+            width="200"
+            height="200"
+            class="botcards"
+            alt="An image of a Paper"
+          />
+          <img
+            src="https://static.thenounproject.com/png/88666-200.png"
+            width="200"
+            height="200"
+            class="botcards"
+            alt="An image of a Scissor"
+          />
         </div>
-        <button id="restart" class="restartHide" onclick="restart()">Restart</button>
-
-
+      </div>
+      <button id="restart" class="restartHide" onclick="restartGame()">
+        Restart
+      </button>
     </div>
     <script>
-        const cards = document.getElementById("sGame");
-        const start = document.getElementById("start");
-        const match = document.getElementById("game");
-        const progress = document.getElementById("score");
-        const nameField = document.getElementById("nameField");
-        const restartButton = document.getElementById("restart");
-        const pCards = document.getElementById("pChoices");
-        const bCards = document.getElementById("bChoices");
-        const gBoard = document.getElementById("board");
+      // Constants
+      const kRock = "Rock";
+      const kScissors = "Scissors";
+      const kPaper = "Paper";
 
-        const startState = "START";
-        const playState = "PLAY";
-        const endState = "END";
+      const kStartState = "Start";
+      const kPlayState = "Play";
+      const kEndState = "End";
 
-        let gameState = startState;
+      const kWin = "Win";
+      const kLose = "Lose";
+      const kTie = "Tie";
 
-        let win = 0;
-        let lose = 0;
+      // Elements
+      const cards = document.getElementById("sGame");
+      const startElement = document.getElementById("start");
+      const match = document.getElementById("game");
+      const progress = document.getElementById("score");
+      const nameField = document.getElementById("nameField");
+      const restartButton = document.getElementById("restart");
+      const gBoard = document.getElementById("board");
+      const gameTextElement = document.getElementById("gameText");
+      const commentaryTextElement = document.getElementById("commentaryText");
 
-        function showGame() {
-            gameState = playState;
-            cards.style.display = "block";
-            pCards.style.display = "block";
```

Comment:
you don't need a pCards, and bCards since all you're doing is hiding / showing both. You can just group them into 1 container, and hide / show that container